Source/ACE/Entity/PhysicsData.cs

```diff
@@ -30,10 +30,10 @@ public class PhysicsData
         public float Friction;
         public float Elastcity;
         public uint AnimationFrame;
-        public Position Acceleration;
+        public AceVector3 Acceleration;
```

Comment:
Ok, makes sense.  Just give us a way to get the values out after it's been created.  We can let these vectors live longer than 1 send and just update the values they have in them.  Also, physics engine will need to read them out at some point.